sdk/servicebus/test-resources.json

```diff
@@ -117,6 +181,10 @@
     "SERVICEBUS_SECONDARY_NAMESPACE_CONNECTION_STRING": {
       "type": "string",
       "value": "[listkeys(variables('authRuleSecondaryResourceId'), variables('sbVersion')).primaryConnectionString]"
+    },
+    "STORAGE_CLAIM_CHECK_CONNECTION_STRING": {
```

Comment:
Why not just `SERVICEBUS_STORAGE_CONNECTION_STRING`?

Reply:
I intentionally included CLAIM_CHECK to highlight what this is used for. Unlike Event Hubs, this isn't actually used in the produce code.